producer-c/producer-cloudwatch-integ/KvsProducerSampleCloudwatch.cpp

```diff
@@ -180,26 +197,53 @@ INT32 main(INT32 argc, CHAR *argv[])
             frame.presentationTs = frame.decodingTs;
             frameIndex++;
         }
+        if (STATUS_FAILED(retStatus)) {
+            DLOGE("Failed with status 0x%08x\n", retStatus);
+         }
 
+        if (frame.frameData != NULL) {
+            MEMFREE(frame.frameData);
+        }
+
+        freeDeviceInfo(&pDeviceInfo);
+        freeStreamInfoProvider(&pStreamInfo);
+        freeKinesisVideoStream(&streamHandle);
+        freeKinesisVideoClient(&clientHandle);
+        freeCallbacksProvider(&pClientCallbacks); // This will also take care of freeing canaryStreamCallbacks
+        DLOGI("Memory allocated:%llu bytes", getInstrumentedTotalAllocationSize());
```

Comment:
No need to print this line. The reset_instrumented_allocators will printout if the total memory is not zero

Reply:
Removed